xray/run.py

```diff
@@ -13,16 +13,16 @@
 
 def run_command(cmd, capture_output=False):
     args = shlex.split(cmd)
-    run = subprocess.run(args, capture_output=capture_output,check=True)
+    run = subprocess.run(args, capture_output=capture_output, check=True)
     return (run.stdout, run.stderr)
 
 
 def get_csv_name(wg, test_type, count):
     return f"results/xray_metrics_{wg.lower()}_{test_type}_{count}.csv"
 
 
-def get_pcap_name(wg, test_type, count):
-    return f"results/{WG_IFC_NAME}_{wg.lower()}_{test_type}_{count}.pcap"
+def get_test_path(wg, test_type, count):
```

Comment:
I think I would prefer to have a separate function for getting the output files you're creating for now. The naming scheme I started here is not reusable enough (the images you output would be named `xraywg1*` which is the name of the interface). I think a bit of an overhaul of these path getters would be good and then we could centralize on a single function for this

Reply:
Good idea, refactored this into a separate module